### DISCUSSION TITLE
HttpClient Support for AWS Java SDK

### DISCUSSION THREAD

**Discussion by vnilawar** [aws/aws-sdk-java#2780] [Q&A] [answered]
I have been using AWS Java SDK(aws-java-sdk-core & aws-java-sdk-s3) 1.12.78 alongside (org.apache.httpcomponents httpclient 4.5.13, Java11) and the default version that AWS Java SDK comes with is 4.5.13

Httpclient.jar is 4.x version will soon be out of support, we plan to use httpclient5 5.1.3
Can we get support for httpclient5 for AWS Java SDK?

**Comment by debora-ito** [comment, marked as answer]
Hi @vnilawar thank you for reaching out.

The Java SDK v1 is not getting updates or new features, the team is focused in releasing features for Java SDK v2 right now. We don't have plans to upgrade Apache httpclient, but you can overwrite the httpclient version in your own application.
